pipeline-scripts/workload.groovy

```diff
@@ -75,7 +75,7 @@ stage (workload) {
 
 	// OpenShift install specific logic 
 	// Install cluster using the payload captured at the build trigger url when scale_ci_build_trigger is set
-	if ( workload.toString().trim().equals("ATS-SCALE-CI-OCP-AWS-DEPLOY") || workload.toString().trim().equals("ATS-SCALE-CI-OCP-GCP-DEPLOY") || workload.toString().trim().equals("ATS-SCALE-CI-OCP-AZURE-DEPLOY") || workload.toString().trim().equals("ATS-SCALE-CI-OCP-OSP-DEPLOY") ) {
+	if ( workload.toString().trim().equals("ATS-SCALE-CI-OCP-AWS-DEPLOY") || workload.toString().trim().equals("ATS-SCALE-CI-OCP-GCP-DEPLOY") || workload.toString().trim().equals("ATS-SCALE-CI-OCP-AZURE-DEPLOY") || workload.toString().trim().equals("ATS-SCALE-CI-OCP-OSP-DEPLOY") || workload.toString().trim().equals("ATS-SCALE-CI-OCP-BM-DEPLOY") ) {
```

Comment:
We might want to replace the parameter - https://github.com/cloud-bulldozer/scale-ci-pipeline/blob/95e82cde01cbc4d70d35ca5bc29fe96396f28e3c/pipeline-scripts/workload.groovy#L90 with the one Jetski uses to define the build to install - OPENSHIFT_RELEASE.

Reply:
good catch